build.gradle

```diff
@@ -1104,33 +1127,24 @@ project(':generator') {
 project(':clients') {
   archivesBaseName = "kafka-clients"
 
-  configurations {
-    jacksonDatabindConfig
-  }
-
-  // add jacksonDatabindConfig as provided scope config with high priority (1000)
-  conf2ScopeMappings.addMapping(1000, configurations.jacksonDatabindConfig, "provided")
-
   dependencies {
-    compile libs.zstd
-    compile libs.lz4
-    compile libs.snappy
-    compile libs.slf4jApi
+    implementation libs.zstd
```

Comment:
How about exposing those compression libraries for compatibility?

Reply:
Applications should not have a compile time dependency to these libraries, right? They would have a runtime dependency.

Reply:
you are right :)